src/ui/view/share/data/tagsField/tagsField.scss

```diff
@@ -0,0 +1,62 @@
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-tags-field-main {
+  $tag-height: 28px;
+  @include with-theme() {
+    box-sizing: border-box;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 10px;
+    padding: 10px;
+    max-height: 86px;
+    overflow-y: auto;
+    width: 100%;
+    border-radius: 10px;
+    background-color: themed('card');
+
+    .okp4-dataverse-portal-tag-main {
+      min-height: $tag-height;
+    }
+
+    .okp4-dataverse-portal-tags-field {
+      position: relative;
+      height: $tag-height;
+
+      &-placeholder {
```

Comment:
This way to handle classnames works fine, but to stay in line with the way we manage classnames in general, can you keep our original way of doing things?